test/CMakeLists.txt

```diff
@@ -15,6 +15,14 @@ target_compile_options(
 		-std=c11
```

Comment:
Is it necessary to make changes to all of the CMakeLists files?

Reply:
I needed to do this in order to fully instrument the library for fuzzing. If I omitted this, I wasn't getting proper instrumentation across the codebase